pexec/process.go

```diff
@@ -5,16 +5,21 @@
 // when configured.
 package pexec
 
-import "go.viam.com/utils"
+import (
+	"io"
+
+	"go.viam.com/utils"
+)
 
 // A ProcessConfig describes how to manage a system process.
 type ProcessConfig struct {
-	ID      string   `json:"id"`
-	Name    string   `json:"name"`
-	Args    []string `json:"args"`
-	CWD     string   `json:"cwd"`
-	OneShot bool     `json:"one_shot"`
-	Log     bool     `json:"log"`
+	ID        string     `json:"id"`
+	Name      string     `json:"name"`
+	Args      []string   `json:"args"`
+	CWD       string     `json:"cwd"`
+	OneShot   bool       `json:"one_shot"`
+	Log       bool       `json:"log"`
+	LogWriter *io.Writer `json:"-"`
```

Comment:
you don't need a pointer

Reply:
How else can I make this optional?

Reply:
an interface is still able to be nil, so you check that